microservices/product-search/README.md

```diff
@@ -53,7 +53,7 @@ Some attributes of a product can be different across the domains.
 
 Example:
 ```php
-$this->microserviceProductSearchClient->get('product-ids', [
+$this->microserviceProductSearchClient->get('search-product-ids', [
```

Comment:
could you make this change in an appropriate commit? (I think the first or second one it is)

Reply:
Done :)